### PR TITLE
Competitors Joining Late in FMC Multi-Location Competitions

### PR DESCRIPTION
**Pull request by Nanush7**
Competitors must not join an FMC attempt after it has started at multi location competitions. This proposal also moves most of E2b++ to a new regulation (E6).

### CODE REVIEW

**Comment by Nanush7** [comment]
I think this is relevant enough to move it to a regulation. But it's just an idea.

File: wca-regulations.md [hunk 1]
```diff
@@ -528,6 +528,10 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - E3f) Tools to remove markings (self-supplied), such as correction fluid, correction tape, or erasers.
 - E4) The WCA Delegate may choose to publish the solutions (e.g. photo of the score sheets or transcriptions of solutions).
 - E5) The competitor must not communicate with anyone other than the WCA Delegate or the judge. This communication should be about the WCA Regulations and the competitor must not receive assistance deriving their solution. Penalty: disqualification of the attempt (DNF).
+- E6) A competitor may join a 3x3x3 Fewest Moves attempt that is already in progress, at the discretion of the WCA Delegate. Exception: a competitor must not be permitted to join an ongoing attempt in the following cases:
+    - E6a) A competitor has finished their attempt already (see [Guideline E2b+](guidelines:guideline:E2b+)).
+    - E6b) The attempt takes place in a competition with multiple physical locations (e.g. "3x3x3 Fewest Moves simultaneous competitions").
```
Comment (ohexter) [inline wca-regulations.md]: Should "e.g." intead be "i.e."? (also applies to 9u).

Comment (Nanush7) [reply wca-regulations.md]: Makes sense!

**Comment by CarterKoala** [comment]
Any reason not to move the rest of E2b++ to the regulation?

**Comment by Nanush7** [comment]
I was thinking about leaving it there because it's mostly relevant to E2b. But either way would be fine IMO.

**Comment by CarterKoala** [comment]
I think it would be fine with either E2b or E6, but it doesn't seem right to be a guideline and not a reg.

**Comment by Nanush7** [comment]
João proposed adding a reasonable delay (~5 min) as in FMC 2024. Posting here so the idea doesn't get lost.

**Pull request merged**